src/main/kotlin/no/nav/amt/person/service/clients/veilarboppfolging/VeilarboppfolgingClient.kt

```diff
@@ -41,7 +44,28 @@ class VeilarboppfolgingClient(
 
 			return veilederRespons.veilederIdent
 		}
+	}
+
+	fun hentOppfolgingperioder(fnr: String) : List<Oppfolgingsperiode> {
+		val personRequestJson = toJsonString(PersonRequest(fnr))
+		val request = Request.Builder()
+			.url("$apiUrl/api/v3/oppfolging/hent-status")
+			.header("Accept", "application/json; charset=utf-8")
+			.header("Authorization", "Bearer ${veilarboppfolgingTokenProvider.get()}")
+			.post(personRequestJson.toRequestBody(mediaTypeJson))
+			.build()
 
+		httpClient.newCall(request).execute().use { response ->
+			response.takeIf { !it.isSuccessful }
```

Comment:
Vet ikke om jeg er fan av å bruke take i steden for å bare skrive en if 😅
Hvis man bruker det man tar så er det noe annet

Reply:
Denne her la jeg ikke veldig mye tanke i, den er bare kopiert fra apikallet rett over 😅  Jeg pleier også å bruke en if for dette her. Kan endre det :)